app/src/main/java/com/github/se/gomeet/ui/mainscreens/events/posts/EventPost.kt

```diff
@@ -63,14 +69,17 @@ fun EventPost(
     post: Post,
     userViewModel: UserViewModel,
     eventViewModel: EventViewModel,
-    currentUser: String
+    currentUser: String,
```

Comment:
currentUser shouldn't be passed as a parameter anymore if there is an EventViewModel or a UserViewModel passed as well. Both viewmodels have access to the current user via ...ViewModel.currentUID!! which is guaranteed to be non-null as long as it is the instance passed from InitFunctions.

Reply:
I just used the `EventPost` function that was implemented in a previous PR by another developper.

Reply:
My bad, it showed up as a modification in the git differences and at this particular difference a bit quickly and so I thought you had added it yourself 😅